xsd-fu/python/util.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
Same as above, should this be included in #202?

Reply:
I cherry-picked the commit from #202.